activesupport/lib/active_support/testing/parallelization.rb

```diff
@@ -63,24 +63,28 @@ def run_cleanup(worker)
       end
 
       def start
-        @pool = @queue_size.times.map do |worker|
-          fork do
-            DRb.stop_service
-
-            after_fork(worker)
-
-            queue = DRbObject.new_with_uri(@url)
-
-            while job = queue.pop
-              klass    = job[0]
-              method   = job[1]
-              reporter = job[2]
-              result   = Minitest.run_one_method(klass, method)
-
-              queue.record(reporter, result)
+        begin
```

Comment:
Is this `begin` without `ensure` needed?

Reply:
Oops fixed that.